tests/socket_utils_tests.py

```diff
@@ -0,0 +1,43 @@
+# NEON AI (TM) SOFTWARE, Software Development Kit & Application Development System
+#
+# Copyright 2008-2021 Neongecko.com Inc. | All Rights Reserved
+#
+# Notice of License - Duplicating this Notice of License near the start of any file containing
+# a derivative of this software is a condition of license for this software.
+# Friendly Licensing:
+# No charge, open source royalty free use of the Neon AI software source and object is offered for
+# educational users, noncommercial enthusiasts, Public Benefit Corporations (and LLCs) and
+# Social Purpose Corporations (and LLCs). Developers can contact developers@neon.ai
+# For commercial licensing, distribution of derivative works or redistribution please contact licenses@neon.ai
+# Distributed on an "AS IS” basis without warranties or conditions of any kind, either express or implied.
+# Trademarks of Neongecko: Neon AI(TM), Neon Assist (TM), Neon Communicator(TM), Klat(TM)
+# Authors: Guy Daniels, Daniel McKnight, Regina Bloomstine, Elon Gasper, Richard Leeds
+#
+# Specialized conversational reconveyance options from Conversation Processing Intelligence Corp.
+# US Patents 2008-2021: US7424516, US20140161250, US20140177813, US8638908, US8068604, US8553852, US10530923, US10530924
+# China Patent: CN102017585  -  Europe Patent: EU2156652  -  Patents Pending
+import os
+import sys
+import unittest
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
+from neon_utils.socket_utils import *
+
+TEST_DICT = {b"section 1": {"key1": "val1",
+                            "key2": "val2"},
+             "section 2": {"key_1": b"val1",
+                           "key_2": f"val2"}}
+
+
+class SocketUtilsTest(unittest.TestCase):
+    b64_str = None
+
+    def test_01_dict_to_b64(self):
+        self.__class__.b64_str = dict_to_b64(TEST_DICT)
+        self.assertIsInstance(self.__class__.b64_str, bytes)
+        self.assertTrue(len(self.__class__.b64_str) > 0)
+
+    def test_02_b64_to_dict(self):
+        result_dict = b64_to_dict(self.b64_str)
```

Comment:
Consider hard-coding `b64_str` like `TEST_DICT` to avoid `b64_to_dict` depending on `dict_to_b64`